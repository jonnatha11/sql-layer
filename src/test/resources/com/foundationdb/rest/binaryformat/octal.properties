fdbsql.postgres.jsonbinary_output	octal