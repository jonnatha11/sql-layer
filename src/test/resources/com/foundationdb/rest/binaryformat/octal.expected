[
{"id":0,"bin1":"\001\002\003\004","bin2":"\011"}
]